### ISSUE TITLE
How to print the loss without the regularization term during training?

### ISSUE THREAD

**Issue by benjaminklein** [fchollet/keras#2575]
I'm using Keras to train a model with regularization.
On each epoch, the loss is printed but it's the sum of both the loss and regularization term. How can I get it to print those separately instead of their sum?

**Comment by fchollet** [comment]
You add the loss as a metric, e.g.:

from keras import metrics
model.compile(..., metrics=[metrics.categorical_crossentropy])

**Comment by motikadosh** [comment]
Wouldn't that (putting the loss function as a metric also) cause duplicate calculation of the metric?

Perhaps it would be better if the regularization losses would have their own standard term in `logs` dict, since otherwise it's pretty confusing.
Especially when training multiple tasks- the progbar prints each task-loss and I expected the `loss` to be the sum, but it wasn't, it was the sum of all the losses and the regularization loss.
Thanks

**Comment by fzyzcjy** [comment]
any updates?
